test/libsinsp_e2e/sys_call_test.cpp

```diff
@@ -1311,3 +1311,106 @@ TEST_F(sys_call_test, unshare_)
 	ASSERT_NO_FATAL_FAILURE({ event_capture::run(test, callback, filter); });
 	EXPECT_EQ(2, callnum);
 }
+
+TEST_F(sys_call_test, sendmsg_recvmsg_SCM_RIGHTS)
```

Comment:
@therealbobo can you give it a check? This test is the one that triggers the use-after-free to avoid future issues ;)

Reply:
LGTM! 😄